R/functions-Chromatograms.R

```diff
@@ -0,0 +1,104 @@
+## functions for Chromatograms class
+
+.validChromatograms <- function(x) {
+    msg <- character()
```

Comment:
I had the `Biobase::validMsg` calls in the validation function of the classes in https://github.com/Bioconductor/AnnotationFilter , but Martin removed them all :-)